lib/timer_face/PieTimer.dart

```diff
@@ -86,30 +86,38 @@ class _PieTimerState extends State<PieTimer> with TickerProviderStateMixin {
 
   // creates the main circle graphic
   void generatePie() {
-    // Map<String, double> dataMap = new Map();
-    // dataMap.putIfAbsent("Mobile Apps Mockings", () => 10);
-    // dataMap.putIfAbsent("Graphic Design Sketch", () => 15);
-    // dataMap.putIfAbsent("Core 250 RR", () => 10);
     Widget pie = Positioned.fill(
       child: CustomPaint(
           painter: CustomTimerPainter(
-              animation: _controller, color: Colors.blueGrey[800])),
+              animation: _controller, color: Colors.blueGrey[700])),
```

Comment:
How are we differentiating between colors used from the theme versus the adhoc colors used?